src/Lookup/SiteLinkLookup.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace Wikibase\DataModel\Services\Lookup;
+
+use Wikibase\DataModel\Entity\ItemId;
+use Wikibase\DataModel\SiteLink;
+
+/**
+ * Contains methods to lookup of sitelinks of lookup by sitelinks.
+ *
+ * @since 2.0
+ *
+ * @licence GNU GPL v2+
+ * @author Jeroen De Dauw < jeroendedauw@gmail.com >
+ */
+interface SiteLinkLookup {
+
+	/**
+	 * Returns the id of the item that is equivalent to the
+	 * provided page, or null if there is none.
+	 *
+	 * @since 2.0
+	 *
+	 * @param string $globalSiteId
+	 * @param string $pageTitle
```

Comment:
Similar problem here. Why does this use two strings instead of a SiteLink object?